client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/product/DomainsTransferredList.tsx

```diff
@@ -0,0 +1,54 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useI18n } from '@wordpress/react-i18n';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+import './style.scss';
+
+type Props = {
+	purchases: ReceiptPurchase[] | undefined;
+};
+
+const DomainsTransferredList = ( { purchases }: Props ) => {
+	const { __, _n } = useI18n();
+
+	const handleUserClick = ( destination: string ) => {
+		recordTracksEvent( 'calypso_domain_transfer_complete_click', {
+			destination,
+		} );
+	};
+
+	return (
+		<>
+			<div className="domain-header-buttons">
+				<a
+					href="/setup/domain-transfer"
+					onClick={ () => handleUserClick( '/setup/domain-transfer' ) }
+					className="components-button is-secondary"
+				>
+					{ __( 'Transfer more domains' ) }
+				</a>
+
+				<a
+					href="/domains/manage?filter=owned-by-me&sortKey=registered-until"
+					className="components-button is-primary manage-all-domains"
+					onClick={ () => handleUserClick( '/domains/manage' ) }
+				>
+					{ _n( 'Manage your domain', 'Manage your domains', purchases?.length ?? 0 ) }
+				</a>
+			</div>
```

Comment:
@candy02058912 this sort of flow specific header is going to be a problem for our unified thank you page design. 

We need to make a decision between a unified checkout receipt page and custom flow completion pages.

I'm wondering if maybe these thank you pages really belong in the stepper framework (e.g. marketplace approach). That way the flow can determine if generic thank you page is ok or if it wants to do something extra? I'd kinda prefer not to do this but if we keep getting flow specific designs I don't see another option.

I don't know what that means for non stepper checkout flows though, maybe the checkout belongs in stepper too? I don't think so... Maybe we need to go back to something like the "new" ThankYou component and deal with the fact there will never be a unified thank you page?